tests/lib/mongo/fixtures/buckets.fixtures.ts

```diff
@@ -0,0 +1,37 @@
+import { Bucket } from "../../../../lib/core/buckets/Bucket";
+import { ObjectId } from "mongodb";
+
+type MongoBucketModel = Required<Omit<Bucket, "id">> & {
+  _id: ObjectId;
+};
+
+const userOneBuckets: MongoBucketModel[] = [
+  {
+    _id: new ObjectId("72b814bf3cde6dcc6f6c9a7b"),
+    user: "user@user.com",
```

Comment:
Try to use variables instead of hardcoded values, for instance:

`export const userOneEmail = "user@user.com";`

And then use it wherever you want, thus reducing the failure rate